### ISSUE TITLE
No data available in table

### ISSUE THREAD

**Issue by major697** [PluginsOCSInventory-NG/officepack#45]
Hello I installed officepack on my Centos server, I used this instructions:

http://ask.ocsinventory-ng.org/8236/plugin-officepack-for-ocsserver2-3-1-ubuntu16-04
and
http://ask.ocsinventory-ng.org/4289/officepack-in-ocs-server-2-2-problem-and-solution

I have permission to my file and folder:
**File:** /etc/ocsinventory-server/perl/Apache/Ocsinventory/Plugins/Officepack/Map.pm
**Permission:** -rw-rw-rw-. 1 apache root 1064 08-02 11:19 Map.pm

**File:** /etc/ocsinventory-server/plugins/Officepack.conf
**Permission:** -rw-rw-rw-. 1 apache root   58 08-02 11:17 officepack.conf

**Folder:** /usr/share/ocsinventory-reports/ocsreports/plugins/computer_detail
**Permission:** drwxr-xr-x. 33 apache root   4096 07-19 09:48 computer_detail

I extort get new data from computer to OCS, but still I have empty "Office Key Manager"

My server:
PHP: 5.6.36
Server WWW: httpd
MYSQL: Ver 14.14 Distrib 5.6.10

OCS: ver 2.5
OfficePack: ver 2.0
WindowsAgentOCS: ver 2.3

**Comment by helicrenz** [comment]
Yes having the same problem did you manage to fix your issue? May i know what officepack plugin your using

**Comment by Lea9250** [comment]
Hi @major697 

This issue has been fixed with the most recent versions of the plugin and OCS.

Best regards,
Léa
